vite-project/src/App.jsx

```diff
@@ -1,12 +1,19 @@
-
+import { useState } from 'react'
 import './App.css'
+import Search from './components/Search/Search'
 
+function App() {
+  
+  // eslint-disable-next-line no-unused-vars
+  const [searchData, setSearchData] = useState(""); 
 
+  const handleOnSearchChange = (searchData) => {
+    setSearchData(searchData);
+  }
 
-function App() {
   return (
     <>
- 
+      <Search onSearchChange={handleOnSearchChange} /> 
```

Comment:
add a container components that will include the componnets, and import it here, App.tsx file shouldn't have much logic